src/Tools/BuildActionTelemetryTable/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+
+namespace BuildActionTelemetryTable
+{
+    class Program
+    {
+        private static readonly string s_executingPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+
+        private static readonly string[] s_excludeList = new[]
+        {
+            "Microsoft.CodeAnalysis.AnalyzerUtilities.dll"
+        };
+
+        static void Main(string[] args)
+        {
+            var assemblies = GetAssemblies(args);
+            var codeActionTypes = GetCodeActionTypes(assemblies);
+            var telemetryInfos = codeActionTypes.Select(type => GetTelemetryInfo(type));
+
+            var hashes = new StringBuilder();
+
+            hashes.AppendLine("let actions = datatable(ActionName: string, Prefix: string, Suffix: string)");
```

Comment:
It would be good to note that this is outputting the Kusto datatable construction that can be used in Kusto queries.

Reply:
@ryzngard Added logging based on your suggestion.